src/main/webapp/definitions/alfresco-search.yaml

```diff
@@ -335,6 +332,11 @@ paths:
                   ]
               },
         ```
+        **Note**: after the migration to Swagger UI 3, this API definition is raising **two errors on the top of this page**, related to the `pivots` field.
```

Comment:
This is great, but personally, I would put this right at the beginning of the description so it's the first thing the user reads.